src/components/device-page/reporting-row.tsx

```diff
@@ -95,10 +95,10 @@ export class ReportingRow extends Component<ReportingRowProps & WithTranslation,
         this.setState({ stateRule });
     }
 
-    changeHandler = (event: ChangeEvent<HTMLInputElement>): void => {
+    changeHandlerNumber = (event: ChangeEvent<HTMLInputElement>): void => {
         const { stateRule } = this.state;
         const { name, value } = event.target;
-        stateRule[name] = value;
+        stateRule[name] = Number(value);
```

Comment:
it would be better to use valueAsNumber
```
-        const { name, value } = event.target;
-        stateRule[name] = value;
+        const { name, valueAsNumber } = event.target;
+        stateRule[name] = valueAsNumber;
```

Reply:
Clean! Didn't know about this yet.